ConsoleWrapper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FrenchApp
+{
+    internal class ConsoleWrapper : IInOutHandler
```

Comment:
The name of this interface is OK. Actually, it is well designed, but its methods are not. You are lacking implementation details (_of console out_).